app/src/main/java/org/oppia/app/walkthrough/WalkthroughFragmentChangeListener.kt

```diff
@@ -0,0 +1,13 @@
+package org.oppia.app.walkthrough
+
+/** Represents current state of Walkthrough Fragment. */
+enum class WalkthroughPages(val value: Int) {
+  WELCOME(0),
+  TOPICLIST(1),
+  FINAL(2)
+}
+
+/** Listener for when an activity should change a fragment position. */
+interface WalkthroughPageChangeListener {
```

Comment:
Please create enum and interface in separate files for this case.

Reply:
@rt4914 Can you tell which changes because all of the above-requested changes had already been done.

Reply:
Commented on some pending changes.

Also, please follow these steps that I had mentioned earlier on the PR: https://github.com/oppia/oppia-android/pull/730#pullrequestreview-370143394

As this helps to make a better PR.